package.json

```diff
@@ -17,6 +17,7 @@
   "author": "Mapbox",
   "license": "ISC",
   "dependencies": {
+    "mason-js-sdk": "https://github.com/mapbox/mason-js/tarball/fix-linking-a-link",
```

Comment:
@millzpaugh Could you give a review on https://github.com/mapbox/mason-js/pull/48? If that looks good, would be ideal to publish a new mason-js so this can be updated to use a formal release.